packages/flutter_tools/lib/executable.dart

```diff
@@ -66,6 +67,7 @@ Future<Null> main(List<String> args) async {
     ..addCommand(new RunCommand())
     ..addCommand(new RunMojoCommand(hidden: !verboseHelp))
     ..addCommand(new ScreenshotCommand())
+    ..addCommand(new SkiaCommand())
```

Comment:
You don't think end-developers will be interested in looking at this information to figure out how to optimize the performance of their apps?